docs/explanations.rst

```diff
@@ -25,7 +25,7 @@ How Craft CLI manage the application logs
 
 Unless overridden when ``emit`` is initiated (see :ref:`how to do that <howto_other_logfile>`), the application logs will be managed by the Craft CLI library, according to the following rules:
 
-- one log file is always produced for each application run (only exposed to the user if the application ends in error or a verbose run was requested by ``--verbose`` or ``--trace``), naming the files with a timestamp so they are unique
+- one log file is always produced for each application run (only exposed to the user if the application ends in error or a verbose run was requested by ``--verbose`` or ``--verbosity=trace``), naming the files with a timestamp so they are unique
```

Comment:
what about --verbosity=debug ?

Reply:
I was considering it an example... I'll be more explicit and specify it as an example. I don't want to be 100% complete, otherwise I'll need to also mention --verbosity=verbose, and it will be a mess regarding readability.